src/Alexa.Net/project.json

```diff
@@ -6,11 +6,11 @@
     "Newtonsoft.Json": "9.0.1"
   },
   "frameworks": {
-    "netstandard1.4": {}
+    "netstandard1.2": {}
```

Comment:
To my knowledge .NetStandard 1.6 does not support .NetFull. .NetStandard 1.2 is the highest version to support PCL (it's equivalent to Profile151). I am trying to maintain the most compatibility with the most people. 

Also https://docs.microsoft.com/en-us/dotnet/articles/core/packages

> It may seem strange to target netstandard1.3 but use the 1.6.0 version of NETStandard.Library. It is a valid use-case, since the metapackage maintains support for older netstandard versions. It could be the case you've standardized on the 1.6.0 version of the metapackage and use it for all your libraries, which target a variety of netstandard versions. With this approach, you only need to restore NETStandard.Library 1.6.0 and not earlier versions.